discord/automod.py

```diff
@@ -114,27 +114,47 @@ def to_dict(self) -> Dict[str, Any]:
 
 
 class AutoModTrigger:
-    """Represents a trigger for an auto moderation rule.
+    r"""Represents a trigger for an auto moderation rule.
+
+    The following table illustrates relevant attributes for each :class:`AutoModRuleTriggerType`:
+
+    +-----------------------------------------------+--------------------------------------+
+    |                    Type                       |              Attributes              |
+    +===============================================+======================================+
+    | :attr:`AutoModRuleTriggerType.keyword`        | :attr:`keyword_filter`               |
+    +-----------------------------------------------+--------------------------------------+
+    | :attr:`AutoModRuleTriggerType.harmful_link`   |                                      |
```

Comment:
This trigger type is apparently deprecated/removed and is on by default according to DAPI docs so it might be worth removing this before pypi release